synfig-core/src/synfig/layers/layer_bitmap.h

```diff
@@ -89,6 +89,8 @@ class Layer_Bitmap : public Layer_Composite, public Layer_NoDeform
 	virtual Rect get_bounding_rect()const;
 
 	virtual synfig::Layer::Handle hit_check(synfig::Context context, const synfig::Point &point)const;
+
+	bool is_same_as(Layer_Bitmap::Handle bitmap);
```

Comment:
So is this useless now?

Reply:
Yes! I thought I removed it 😬 Will do that!